data/datacatalogue/CatalogueOntologies/Network features.csv

```diff
@@ -1,41 +1,42 @@
-name,parent,definition
-ENCePP network,,Do you want to become an ENCePP partner?
-Primary therapeutic area,,Please specify the primary therapeutic/disease area of your Network. Multiple values can be selected.
-Anaesthesia,Primary therapeutic area,
-Cardiovascular diseases ,Primary therapeutic area,
-Congenital malformations,Primary therapeutic area,
-Devices,Primary therapeutic area,
-Disorders of the central nervous system,Primary therapeutic area,
-"Ear, nose and oropharynx disorders",Primary therapeutic area,
-Endocrine disorders,Primary therapeutic area,
-Eye disorders ,Primary therapeutic area,
-Gastrointestinal tract,Primary therapeutic area,
-Geriatrics,Primary therapeutic area,
-Gynaecology,Primary therapeutic area,
-Immunological products and vaccines,Primary therapeutic area,
-Immunosuppression Infectious diseases,Primary therapeutic area,
-Liver disease,Primary therapeutic area,
-Malignant disease,Primary therapeutic area,
-Musculoskeletal and joint diseases,Primary therapeutic area,
-Neonates,Primary therapeutic area,
-Nutrition and blood,Primary therapeutic area,
-Osteoporosis,Primary therapeutic area,
-Paediatrics,Primary therapeutic area,
-Poisoning/Overdose,Primary therapeutic area,
-Pregnancy,Primary therapeutic area,
-Psychiatry,Primary therapeutic area,
-Renal impairment,Primary therapeutic area,
-Respiratory diseases,Primary therapeutic area,
-Skin disorders,Primary therapeutic area,
-Urinary tract disorders,Primary therapeutic area,
-Other,Primary therapeutic area,
-Funding,,Please specify the type of funding for the Network. Multiple values can be selected.
-EU institutional research programme,Funding,
-non-EU institutional research programme,Funding,
-EMA,Funding,
-national competent authority (NCAs),Funding,
-other public funding (e.g.: hospital,Funding,
-non for-profit organisation (e.g.,Funding,
```

Comment:
non for-profit organisation (e.g.,Funding, is now spelled:
Non for-profit organisation (e.g. charity)